decaflex/contrib/evaluate.sh

```diff
@@ -0,0 +1,12 @@
+FILES=references/dev/*.out
+for filename in $FILES; do 
+	IFS='/' read -r -a file <<< "$filename" 
+	REFERENCE=references/dev/${file[2]} 
+	OUTPUT=output/dev/${file[2]} 
+	DIFF=$(diff $REFERENCE $OUTPUT) 
```

Comment:
please add `-w` option to the diff command so the output of this file matches the outputs in `check.py`